templates/home.html

```diff
@@ -1,35 +1,44 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:query {"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"layout":{"inherit":true},"tagName":"main"} -->
-<main class="wp-block-query"><!-- wp:post-template {"align":"wide"} -->
-	<!-- wp:post-title {"isLink":true,"align":"wide"} /-->
-
-	<!-- wp:post-featured-image {"isLink":true,"align":"wide"} /-->
-
-	<!-- wp:columns {"align":"wide"} -->
-	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"650px"} -->
-	<div class="wp-block-column" style="flex-basis:650px"><!-- wp:post-excerpt /-->
-
-	<!-- wp:post-date {"isLink":true,"format":"F j, Y"} /--></div>
-	<!-- /wp:column -->
-
-	<!-- wp:column {"width":""} -->
-	<div class="wp-block-column"></div>
-	<!-- /wp:column --></div>
-	<!-- /wp:columns -->
-
-	<!-- wp:spacer {"height":112} -->
-	<div style="height:112px" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-	<!-- /wp:post-template -->
-
-	<!-- wp:query-pagination {"paginationArrow":"arrow","align":"wide","layout":{"type":"flex","justifyContent":"space-between"}} -->
-	<!-- wp:query-pagination-previous /-->
-
-	<!-- wp:query-pagination-numbers /-->
-
-	<!-- wp:query-pagination-next /-->
-	<!-- /wp:query-pagination --></main>
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"8rem","bottom":"8rem"}}},"layout":{"inherit":true}} -->
+<main class="wp-block-group" style="margin-top:8rem;margin-bottom:8rem">
```

Comment:
I did not find a matching spacing preset for this margin.

Reply:
I've just updated the spacing presets in another PR: https://github.com/WordPress/twentytwentythree/pull/52

I'm not sure if what I've done in that PR is what we should go with, but if we go down that route, there is a spacing preset that would fit `8rem` there.